source/dyaml/test/representer.d

```diff
@@ -24,7 +24,7 @@ import dyaml.test.constructor;
 ///          codeFilename = File name to determine test case from.
 ///                         Nothing is read from this file, it only exists
 ///                         to specify that we need a matching unittest.
-void testRepresenterTypes(bool verbose, string codeFilename)
+void testRepresenterTypes(bool verbose, string codeFilename) @trusted
```

Comment:
better mark the unittest as `@system` is this isn't safe yet.

Reply:
The only thing `@system` about this function is a cast(string)ubyte[], and the result of the cast is only used if the tests are already failing. I don't believe there is any chance for corruption or undefined behaviour here.

Reply:
My point was that you have a test that is @safe to ensure that the inference works correctly, but then you set the wrapping tester to `@trusted`.